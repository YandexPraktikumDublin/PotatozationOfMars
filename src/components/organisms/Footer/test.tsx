import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Footer } from '.'

describe('<Footer />', () => {
  it('should renders correct <Footer />', () => {
    const wrapper = shallow(<Footer />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
