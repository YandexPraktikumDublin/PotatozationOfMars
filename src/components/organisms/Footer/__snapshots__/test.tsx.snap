// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should renders correct <Footer /> 1`] = `
<footer
  className="p-2 items-center flex"
>
  <button
    className="ml-auto mr-4"
  >
    <img
      alt=""
      src={[Function]}
    />
  </button>
</footer>
`;
